direct/io-kafka/src/test/java/cz/o2/proxima/direct/kafka/KafkaAccessorTest.java

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright 2017-2020 O2 Czech Republic, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.direct.kafka;
+
+import static org.mockito.Matchers.anyCollectionOf;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import cz.o2.proxima.repository.AttributeFamilyDescriptor;
+import cz.o2.proxima.repository.EntityDescriptor;
+import cz.o2.proxima.storage.AccessType;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.ConfigEntry;
+import org.apache.kafka.clients.admin.DescribeConfigsResult;
+import org.apache.kafka.common.KafkaFuture;
+import org.apache.kafka.common.config.ConfigResource;
+import org.apache.kafka.common.config.TopicConfig;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mockito;
+
+public class KafkaAccessorTest {
+
+  // mocks
+  private static AdminClient adminClient;
+  private static KafkaFuture kafkaFuture;
+  private static Map map;
+  private static DescribeConfigsResult cfgResult;
+  private static AttributeFamilyDescriptor attrFmlDesc;
+  private static AccessType accessType;
+  private static List<Config> cfgs;
+  private static KafkaAccessor kafkaAccessor;
+  private static List<ConfigEntry> cfgEtrs;
+
+  @BeforeClass
+  public static void setupOnlyOnce() throws ExecutionException, InterruptedException {
+    adminClient = Mockito.mock(AdminClient.class);
+    kafkaFuture = mock(KafkaFuture.class);
+    map = mock(Map.class);
```

Comment:
there is not reason to mock Map class here. We can simply construct a new map without mocking.

Reply:
Also I wouldn't setup class level mocks here, as there should be more test, that don't need these.